MANIFEST.SKIP

```diff
@@ -10,6 +10,9 @@
 \.po$
 ^share/[^/]*\.mo$
 ^share/Zonemaster-Backend.pot$
+^share/backend_config.ci_mysql.ini$
+^share/backend_config.ci_postgresql.ini$
+^share/backend_config.ci_sqlite.ini$
```

Comment:
Because these are regular expressions, we should escape the dot characters:
```suggestion
^share/backend_config\.ci_mysql\.ini$
^share/backend_config\.ci_postgresql\.ini$
^share/backend_config\.ci_sqlite\.ini$
```